### DISCUSSION TITLE
本地搜索 no file serach.xml

### DISCUSSION THREAD

**Discussion by emerisly** [YunYouJun/hexo-theme-starry#221] [Q&A 疑难杂症] [answered]
请问大佬一个问题 我配置了5hr+了依然不知道怎么解决
想实现本地搜索功能
现已按照yun教程中的tips关闭engine search并添加local engine，也按照了https://github.com/wzpan/hexo-generator-search 中的说明install hexo-generator-search，并在root的config中添加了search的设置
现在的情况是hexo g后会报错 ./search.xml not found
`Start processing FATAL { err: Error: ENOENT: no such file or directory, open './search.xml' at Object.openSync (node:fs:585:3) at Object.readFileSync (node:fs:453:35) at Hexo.module.exports`
我又试了在https://github.com/wzpan/hexo-generator-search 找到了一个search.xml的template https://github.com/wzpan/hexo-generator-search/blob/master/templates/search.xml 放在了root文件夹中，在这个情况下，本地搜索可以在localhost上能够成功使用搜索功能（如下图）
![image](https://user-images.githubusercontent.com/44758321/147375352-5365a1d1-0c38-42e7-a718-db6598de4312.png)
![image](https://user-images.githubusercontent.com/44758321/147375343-1e6631c4-10f3-40b9-b415-2a4365ad9328.png)
但是hexo deploy后云端未显示任何搜索功能（如下图）
![image](https://user-images.githubusercontent.com/44758321/147375338-4aa55ee3-f85b-4df1-9511-23be9c982132.png)
在把手动加入root文件夹的search.xml删除后，依然会报错
请问我是有什么配置弄错了吗呜呜
感谢！
![image](https://user-images.githubusercontent.com/44758321/147375181-3b6f393a-d0e2-4e50-91fd-34cbe10531c8.png)

**Comment by emerisly** [comment, marked as answer]
芜湖，，，我不理解，，但是我好像deploy成功了
我大不理解。。。现在可以用了，，我什么都没改，，，
麻了，打扰了。
谢谢。

**Reply by emerisly** [reply]
可能是我缓存的问题吧

**Reply by emerisly** [reply]
或者github的server太慢了

**Reply by YunYouJun** [reply]
应该只是 GitHub 缓存刷新问题。
